### PR TITLE
Promote Andrew Wittrock and Dave Walter as new CF-on-K8s leads

### PR DESCRIPTION
**Pull request by gcapizzi**
I am stepping down as the CF-on-K8s WG leads as I'm concentrating on different projects in VMware, and the WG as agreed on @Birdrock and @davewalter as new leads.

### CODE REVIEW

**Comment by beyhan** [comment]
This depends on the outcome of https://github.com/cloudfoundry/community/pull/704

**Comment by ameowlia** [comment]
👋 Hi all, The TOC has accepted [a new RFC around the tech lead promotion process](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0024-promoting-or-revoking-wg-leads.md).

The new process says...

>_The TOC will start a one week final comment period on the PR to collect approvals from the required parties._

👉 We can start this at the next TOC meeting.

>_The TOC MUST approve the PR before it can be merged._

👉 The TOC will approve/reject after the final comment period.

>_At least one current Lead for the affected Working Group SHOULD approve the PR before it can be merged._

✅  Both of the current leads have approved this PR. @georgethebeatle (explicitly) and @gcapizzi (implicitly, by creating the PR).

>_Any current approvers for the affected Working Group MAY 'vote' by commenting "+1" or "-1" on the PR. 75% or more of the votes from current Working Group approvers MUST vote "+1" for the promotion in order for the PR to be merged._

👉 FYI - @cloudfoundry/wg-cf-on-k8s-korifi-approvers You can vote on this PR.

**Comment by beyhan** [comment]
Starting the final comment period on this util next Tuesday 14 of November with the goal to get this approved. P

**Review by stephanme — APPROVED** [review]

**Review by beyhan — APPROVED** [review]

**Review by rkoster — APPROVED** [review]

**Review by ChrisMcGowan — APPROVED** [review]

**Pull request merged**
